app/models/report_composer.rb

```diff
@@ -76,6 +85,10 @@ def send_mail?
       !!mail_to
     end
 
+    def generate_at
+      Time.find_zone("UTC").parse(raw_params['generate_at']) if raw_params['generate_at'].present?
```

Comment:
I have been thinking about what you were writing about the timezone.
For me it is taking server time, which is the same as user time. I have put the time to UTC in API. But do not know what to do in UI, as there it should have been a users time, but I do not know if it is possible to get. Or is it?

Reply:
User can set timezone explicitly in "My account". We default to current browser configuration. I think it should be possible. Remote exec plugin also has scheduling feature which I think respects current user timezone. Perhaps check the code there or ask @adamruzicka who I think worked on this. IIRC the idea was to store everything in UTC but always present it in current user TZ.